Sources/PointFree/Blog/BlogAtomFeed.swift

```diff
@@ -28,7 +29,7 @@ private func feedView(posts: [BlogPost]) -> [Node] {
 
 private func atomEntry(for post: BlogPost) -> AtomEntry {
   return AtomEntry(
-    content: blogPostContentView(post),
+    content: downgrade(node: blogPostContentView(post)),
```

Comment:
need to do a bit of downgrade/upgrade dancing in this PR to accommodate for some shared views that I don't want to convert yet.